x-pack/plugins/alerting/server/plugin.ts

```diff
@@ -139,7 +138,7 @@ export interface AlertingPluginsStart {
 }
 
 export class AlertingPlugin {
-  private readonly config: Promise<AlertsConfig>;
+  private readonly config: AlertingConfig;
```

Comment:
Not sure why this was a Promise, but I am able to get the config synchronously so I updated the code. If there's a reason it has to be a Promise, am happy to revert these changes.